kernels/cross_entropy_loss.py

```diff
@@ -191,13 +230,13 @@ def _cross_entropy_backward(
 
 class Fast_CrossEntropyLoss(torch.autograd.Function):
     @staticmethod
-    def forward(ctx, logits, labels):
+    def forward(ctx, logits, labels, logit_scale = 1.0):
         n_rows, vocab_size = logits.shape
 
         div, mod = divmod(vocab_size, MAX_FUSED_SIZE)
         n_chunks = div + (mod != 0)
         losses = torch.empty(n_rows, dtype = torch.float32, device = "cuda")
-
+        
```

Comment:
nit: drop this

Reply:
Sorry, I don't understand? :)

The extra tab? It's my IDE loves to add those for some reason if so :/

Reply:
Should be fixed now

Reply:
Github managed to add an extra one at the bottom on its own thought lol!

I'll try and fix it tomorrow.

Reply:
No worries! It was just a nit.